src/app/[locale]/(home)/occasions/_components/occasions-table.tsx

```
@@ -0,0 +1,26 @@
+import { Occasion } from "@lib/types/dashboard/occasions";
+import OccasionsRow from "./occasions-row";
+import SearchInput from "./search-input";
```

ESLint

[error] 'SearchInput' is defined but never used.

(@typescript-eslint/no-unused-vars)